docs/source/developing/ci.adoc

```diff
@@ -10,21 +10,51 @@ toc::[]
 [[ci-overview]]
 == Overview
 
-The following sections describes the various CI setups Minishift utilizes.
+Minishift uses several CI build servers such as the Centos CI, AppVeyor CI, Circle CI and the Travis CI. The servers build incoming pull requests, pass it through various test parameters and provide the status of the builds to ensure continuous integration and deployment. Except for Travis CI, they also archive build artifacts of successful builds.
```

Comment:
Can we have this  => Minishift uses several CI such as the Centos, AppVeyor, Circle and the Travis. They build incoming ...